test/python/rest/submit_tests.py

```diff
@@ -0,0 +1,212 @@
+import os
+import sys
+import time
+import unittest
+import logging
+import streamsx.rest
+import streamsx.topology.context
+
+logger = logging.getLogger('submit_tests')
+logger.setLevel(logging.INFO)
+
+
+applicationName = 'TestApplication::Main'
+
+applicationBundle = './output/TestApplication.Main.sab'
+
+applicationParameters = {
+    'iterationInterval': 1.0,
+    'stringParameter': 'Hello, world',
+    'integerParameter': 42,
+    'floatParameter': 3.14159,
+    'booleanParameter': 'true',
+    'listParameter': [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144],
+    'mapParameter': { 'one': 1, 'two': 2, 'three': 3, 'many': sys.maxsize }
+    }
+
+runInterval = 30
+
+def environment_variables_set():
+
+    variables = [ 'STREAMS_INSTALL', 'STREAMING_ANALYTICS_SERVICE_NAME', 'VCAP_SERVICES' ]
+    result = True
+
+    for variable in variables:
+        if variable not in os.environ:
+            logger.error(variable + ' environment variable not set')
+            result = False
+
+    return result
+
+@unittest.skipIf(not environment_variables_set() , "Streams environment variables not set")
+class RestSubmitTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+
+        # temporarily compile test application locally until REST API wrapper supports remote build service 
```

Comment:
Just to note we could use topology to build the app, rather than directly invoking sc.